staging/src/k8s.io/cli-runtime/pkg/genericclioptions/resource/helper.go

```diff
@@ -49,7 +50,7 @@ func NewHelper(client RESTClient, mapping *meta.RESTMapping) *Helper {
 	}
 }
 
-func (m *Helper) Get(namespace, name string, export bool) (runtime.Object, error) {
+func (m *Helper) Get(namespace, name string, export bool) (runtime.Object, []net.WarningHeader, error) {
```

Comment:
I'm inclined to suggest turning those 3 return values into a single struct, wdyt?